w3w-autosuggest/public/js/w3w-autosuggest-public.js

```diff
@@ -30,178 +30,213 @@
 	 * practising this, we should strive to set a better example in our own work.
 	 */
 
-  function createAutosuggestComponent(target, targetParent, targetSibling) {
-    const w3wComponent = document.createElement('what3words-autosuggest')
-
-    w3wComponent.setAttribute('variant', 'inherit')
-    w3wComponent.setAttribute('headers', JSON.stringify({
-      'X-W3W-Plugin':
-        `what3words-Wordpress/${W3W_AUTOSUGGEST_SETTINGS.version} (` + [
-          `PHP/${W3W_AUTOSUGGEST_SETTINGS.php_version}`,
-          `WordPress/${W3W_AUTOSUGGEST_SETTINGS.wp_version}`,
-          `WooCommerce/${W3W_AUTOSUGGEST_SETTINGS.wc_version}`
-        ].join(' ') + ')'
-    }))
-    w3wComponent.setAttribute('api_key', W3W_AUTOSUGGEST_SETTINGS.api_key)
-    w3wComponent.setAttribute('return_coordinates', true)
-    
-    if (!W3W_AUTOSUGGEST_SETTINGS.woocommerce_enabled) {
-      if (W3W_AUTOSUGGEST_SETTINGS.enable_placeholder){
-        target.setAttribute('placeholder', W3W_AUTOSUGGEST_SETTINGS.placeholder)
+  $(function() {
```

Comment:
Looks like a big change but it's because of the indentation change thanks to wrapping in this ready function.
This is needed for line 35 - the `woocommerce_params` object was not always populated by the time this script ran.
The `isPartialWooCommerce` flag is used to represent the scenario mentioned above, where the autosuggest is applied to an existing field.